### PR TITLE
feat: precommit

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
Add precommit through husky + pre-commit (similar to browsertrix-cloud)

### CODE REVIEW

**Comment by ikreymer** [comment]
@Chickensoupwithrice is the pre-commit hook actually running? Wasn't able to get it to run after `yarn install` and then committed something that failed lint.

**Comment by Chickensoupwithrice** [comment]
Ahh, sorry about that, I had the file name wrong. You also will need to run `yarn prepare` to for the `husky` install

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-#!/usr/bin/env sh
```
Comment (ikreymer) [inline .husky/pre-commit:1]: typo here
```suggestion
#!/usr/bin/env sh
```

**Comment by ikreymer** [comment]
yep, i think that fixed it!

**Review by ikreymer — APPROVED** [review]

**Pull request merged**